go.mod

```diff
@@ -205,3 +195,5 @@ require (
 )
 
 replace github.com/onflow/crypto => github.com/onflow/crypto v0.24.9
+
+replace github.com/ethereum/go-etherem => github.com/onflow/go-ethereum v1.13.4
```

Comment:
why is this needed?

Reply:
So it turns out that it's not needed. Removed in https://github.com/onflow/flow-evm-gateway/pull/201/commits/f27b2f135b7926b6d2f0c00ade08a9af2e9caa94